pkg/webui/console/store/middleware/logics/user-preferences.js

```diff
@@ -18,16 +18,82 @@ import createRequestLogic from '@ttn-lw/lib/store/logics/create-request-logic'
 
 import * as userPreferences from '@console/store/actions/user-preferences'
 
+const getBookmarksThroughPagination = async userId => {
+  let page = 1
+  const limit = 100
```

Comment:
```suggestion
  const limit = 1000
```
The max page size of all entities in the backend is 1000. So it's best to use that by default to minimize the number of page fetches.